plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -135,5 +136,10 @@ public void invalidEmptyToolForBaseConfig() throws Exception {
         }
     }
 
+    @Test
+    public void testRootConfiguratorGeneration() throws Exception {
```

Comment:
@timja this is the test your looking for.

Reply:
bit of a stretch to call that a test, would be good to fill it in so that it's failing.

Reply:
Yeah agreed that's not a test just a trial more or less.